leadership.yaml

```diff
@@ -0,0 +1,119 @@
+- type: house
+  title: Speaker the House
+  office: Speaker
+  bioguide: B000589
+  url: http://www.speaker.gov/
+
+- type: house
+  title: Majority Leader
+  office: Majority Leader
+  bioguide: M001165
+  url: http://www.majorityleader.gov/
+
+- type: house
+  title: Majority Whip
+  office: Majority Whip
+  bioguide: S001176
+  url: http://www.majoritywhip.gov/
+
+- type: house
+  title: Republican Conference Chairman
+  office: Republican Conference
+  bioguide: M001159
+  url: http://gop.gov/
+
+- type: house
+  title: Republican Policy Committee Chairman
+  office: Republican Policy Committee
+  bioguide: M001189
+  url: http://policy.house.gov/
+
+- type: house
+  title: Democratic Leader
+  office: Democratic Leader
+  bioguide: P000197
+  url: http://www.democraticleader.gov/
+
+- type: house
+  title: Democratic Whip
+  office: Democratic Whip
+  bioguide: H000874
+  url: http://www.democraticwhip.gov/
+
+- type: house
+  title: Assistant Democratic Leader
+  office: Assistant Democratic Leader
+  bioguide: C000537
+  url: http://assistantdemocraticleader.house.gov/
+
+- type: house
+  title: Democratic Caucus Chairman
+  office: Democratic Caucus
+  bioguide: B000287
+  url: http://www.dems.gov/
+
+- type: house
+  title: Democratic Caucus Vice Chairman
+  office: Democratic Caucus
+  bioguide: C001038
+  url: http://www.dems.gov/
+
+-type: senate
+  title: Majority Leader
+  office: Senate Majority Leader
+  bioguide: M000355
+  url: http://www.mcconnell.senate.gov
+
+- type: senate
+  title: Majority Whip
+  office: Senate Majority Whip
+  bioguide: C001056
+  url: http://www.cornyn.senate.gov
+
+- type: senate
+  title: Republican Conference Chair
+  office: Republican Conference
+  bioguide: T000250
+  url: http://www.republican.senate.gov/
+
+- type: senate
+  title: Republican Policy Committee Chair
+  office: Republican Policy Committee
+  bioguide: B001261
+  url: http://www.rpc.senate.gov/
+
+- type: senate
+  title: Republican Conference Vice Chair
+  office: Republican Conference
+  bioguide: B000575
+  url: http://www.republican.senate.gov/
+
+- type: senate
+  title: Minority Leader
+  office: Senate Minority Leader
+  bioguide: R000146
+  url: http://www.reid.senate.gov
```

Comment:
This URL isn't the "URL of the office of Senate Minority Leader", it's Reid's individual MoC site, so I think is not correct for this field. Keeping the URLs to their respective institutions also keeps this file more static.